components/SignatureCount/index.js

```diff
@@ -0,0 +1,11 @@
+import { SignatureBox } from './styles';
+
+const SignatureCount = props => {
+  return (
+    <ul>
+      <SignatureBox>{props.count} personas</SignatureBox>
+    </ul>
```

Comment:
No sé por qué querríamos un `ul` aquí , si `SignatureBox` es un `h2`, también podríamos actualizar el nombre usando el prefijo Styled.
```suggestion
    <Styled.SignatureCount>{props.count} personas</Styled.SignatureCount>
```

Reply:
la verdad, no sé como se coló el `ul` por ahí, no me he dado cuenta, gracias :)